packages/ui/src/components/Tabs.svelte

```diff
@@ -20,13 +20,15 @@
 
   export let model: TabModel
   export let selected = 0
+  export let hasLeftIndent = false
```

Comment:
I do not think we need to add this to Tab component.

Reply:
Removed